deps/couchbeam-0.8.0/src/couchbeam.erl

```diff
@@ -173,11 +173,14 @@ replicate(#server{options=IbrowseOpts}=Server, RepObj) ->
     Headers = [{"Content-Type", "application/json"}],
     JsonObj = ejson:encode(RepObj),
 
-    case couchbeam_httpc:request_stream({self(), once}, post, Url, IbrowseOpts, Headers,
-            JsonObj) of
-        {ok, ReqId} ->
-            couchbeam_changes:wait_for_change(ReqId);
-        {error, Error} -> {error, Error}
+%%   lookat couchbeam/a7ea218ef147af
+     case couchbeam_httpc:request(post, Url, ["200", "201"], IbrowseOpts,
```

Comment:
@siplabs 
Thoughts on updating couchbeam rather than patching?  We try to not patch deps if at all possible.